### PR TITLE
Improve README.md documentation

### PR DESCRIPTION
**Pull request by ehmicky**
I added documentation for some features that were not previously documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +163,14 @@ By calling `registerPageTrack`, you tell Angulartics to invoke your function on
 
 This is very similar to page tracking. Angulartics will invoke your function every time the event (`analytics-on` attribute) is fired, passing the action (`analytics-event` attribute) and an object composed of any `analytics-*` attributes you put in the element.
 
+If the analytics provider is created async, you can wrap you code with:
+
+    angulartics.waitForVendorApi("var", 1000[, "subvar"], function(window.var) {
```
Comment (lunks) [inline README.md]: typo

File: README.md [hunk 2]
```diff
@@ -208,9 +218,66 @@ Additional properties (for example, category as required by GA) may be specified
 		analytics-event="Download"
 		analytics-category="Content Actions">Download</a>
 
+or setting `analytics-properties`:
+
+	<a href="file.pdf" 
+		analytics-on="click" 
+		analytics-event="Download"
+		analytics-properties="{ category: 'Content Actions' }">Download</a>
```
Comment (lunks) [inline README.md]: :clap:

**Comment by lunks** [comment]
Good job! There are a couple of tweaks here and there, but this improves the README a lot.

File: README.md (hunk 1 above)
Comment (ehmicky) [inline README.md]: Is this "async" or the brackets (I meant them as specifying it is an optional argument)?

Comment (lunks) [inline README.md]: Oh, they are optional arguments. Yeah, it's not very clear you intend it to be an optional argument. Maybe just show an example without the `subvar`?

**Comment by ehmicky** [comment]
I fixed the problematic points.

**Comment by lunks** [comment]
Great job!

**Pull request merged**